content/en/docs/ops/troubleshooting/istioctl-analyze/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Analyze your cluster and local configuration with istioctl analyze
+description: Shows you how to use istioctl analyze to identify potential issues with your configuration.
+weight: 90
+keywords: [istioctl, debugging, kubernetes]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+## Getting started in under a minute
```

Comment:
Before Getting Started, I think you need an intro paragraph or two explaining what this is about in higher level terms.